src/agoradatatools/great_expectations/gx/expectations/gene_info.json

```diff
@@ -430,49 +430,38 @@
         "json_schema": {
           "$id": "http://example.com/example.json",
           "$schema": "https://json-schema.org/draft/2019-09/schema",
-          "default": [],
-          "items": {
-            "default": {},
-            "properties": {
-              "abability_bucket": {
-                "type": "number"
-              },
-              "abability_bucket_definition": {
-                "maxLength": 1000,
-                "minLength": 44,
-                "type": "string"
-              },
-              "classification": {
-                "maxLength": 1000,
-                "minLength": 22,
-                "type": "string"
-              },
-              "pharos_class": {
-                "type": [
-                  "string",
-                  "null"
+          "default": null,
+          "examples": [
```

Comment:
I'm confused why this variable is the only one with "examples". What does that mean?

Reply:
The `examples` are optional to include and just allow you to have written in the JSON schema itself what the data structure should look like. AFAIK this field is not used in any actual computational validation by JSON schema at all